### ISSUE TITLE
Generate columns that are absent in test set, using generative models

### ISSUE THREAD

**Issue by ghost** [eric-yoo/sdc#2]
There are more columns in training set than test set.

For now, we are training the model with columns that resides in both data set.

However, this restricts the chances where we can exploit richer set of data.

I suggest that we train a generative model (possibly [VAE](https://arxiv.org/abs/1606.05908)) to generate missing columns out of existing data.

Thus, we can use these extended records in test time
